README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# hw01
+
+## Alex's Biography 
+
+My name is Alex Lahood. I am from Buffalo, New York. I am a rising 3rd year at UChicago. I am studying Economics. This summer I am taking MACS 30500 as well as working as a research assistant on campus. I enjoy the following things:
+
+* R
+* Summer
+* TV, movies, & games
+  + *One Punch Man*
+  + *Rainbow 6*
+  + *Get Out*
+  + *The Incredibles II*
+
+* The beach
+* All types of music
+
+
+Thank you for coming to my TED Talk. 
+
+[link](www.TED.com)
```

Comment:
Make sure that the link you're adding is **exactly** as the homepage (in this case) that you're trying to link to. In this case, attempting to reach the link: `www.TED.com` automatically redirects to `https://www.ted.com/#/`. Without the correct text, GitHub tried to route to your link _through_ your repository, so it returned an error.